tests/model_registry/model_catalog/utils.py

```
@@ -1086,3 +1082,47 @@ def verify_labels_match(expected_labels: List[Dict[str, Any]], api_labels: List[
+
+    Args:
+        ids (list): List of model set identifiers that correspond to keys in MODELS dict
+
+    Returns:
+        str: YAML formatted catalog configuration with multiple catalog entries
+    """
+    catalog_entries = ""
+
+    for source_id in ids:
+        if source_id not in HF_MODELS:
+            raise ValueError(f"Model ID '{source_id}' not found in MODELS dictionary")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)